WWDC/AppCoordinator.swift

```diff
@@ -262,13 +262,8 @@ final class AppCoordinator {
     }
     
     func receiveNotification(with userInfo: [String : Any]) -> Bool {
-        if liveObserver.processSubscriptionNotification(with: userInfo) {
-            return true
-        } else if RemoteEnvironment.shared.processSubscriptionNotification(with: userInfo) {
-            return true
-        } else {
-            return false
-        }
+        return liveObserver.processSubscriptionNotification(with: userInfo) &&
```

Comment:
Wouldn't this cause both the `liveObserver` and `RemoteEnvironment` methods to be called?

The expected behavior here is that if `liveObserver` handles the notification, `RemoteEnvironment`'s `processSubscriptionNotification` should NOT get called. This may not cause any issues currently, but might cause some issue in the future, especially when we add more classes that need to handle notifications.